modules/installation-special-config-storage.adoc

```diff
@@ -184,6 +190,26 @@ Some other distributions provide Clevis version 17 or later, which use the SHA-2
 You must use a Clevis version that uses SHA-1 to create the thumbprint, to prevent Clevis binding issues when you install {op-system-first} on your {product-title} cluster nodes.
 ====
 
+.. Optional: for offline Tang provisioning, obtain the advertisement from the server using the `curl` command. Replace `http://tang2.example.com:7500` with the URL of your Tang server:
++
+[source,terminal]
+----
+$ curl -f http://tang2.example.com:7500/adv > adv.jws && cat adv.jws
+----
++
+.Expected output
+[source,terminal]
+----
+{"payload": "eyJrZXlzIjogW3siYWxnIjogIkV", "protected": "eyJhbGciOiJFUzUxMiIsImN0eSI", "signature": "ADLgk7fZdE3Yt4FyYsm0pHiau7Q"}
+----
++
+Provide the advertisement file to Clevis for encryption:
++
+[source,terminal]
+----
+$ clevis-encrypt-tang '{"url":"http://tang2.example.com:7500","adv":"adv.jws"}' < /dev/null > /dev/null
+----
```

Comment:
Right now this `clevis-encrypt-tang` command looks like it's part of the the `curl` command step before it. Should they instead be two substeps of the "Optional" step? E.g.,:

```
.. Optional: For offline Tang provisioning:

... Obtain the advertisement from the server using the `curl` command. Replace `\http://tang2.example.com:7500/` with the URL of your Tang server:
+
[source,terminal]
----
$ curl -f http://tang2.example.com:7500/adv > adv.jws && cat adv.jws
----
+
.Expected output
[source,text]
----
{"payload": "eyJrZXlzIjogW3siYWxnIjogIkV", "protected": "eyJhbGciOiJFUzUxMiIsImN0eSI", "signature": "ADLgk7fZdE3Yt4FyYsm0pHiau7Q"}
----

... Provide the advertisement file to Clevis for encryption:
+
[source,terminal]
----
$ clevis-encrypt-tang '{"url":"\http://tang2.example.com:7500/","adv":"adv.jws"}' < /dev/null > /dev/null
----
```
... I recently realized that IBM SG is OK with "A lead-in infinitive phrase (one that starts with _To_)" for procedures (sparingly, since full-sentences are better for translation), so employing that here. 😅 I don't _thiiiink_ it's too weird to have two colons (after "Optional" and after "provisioning). 🤷

Reply:
Good point! I sometimes struggle when  an optional step has multiple parts, because two separate optional steps don't imply a collective optional instruction, which is what matters. making substeps is a good idea 🙂